build.gradle

```diff
@@ -118,8 +118,13 @@ dependencies {
 }
 
 task fatJar(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar) {
-  archiveName = "gapic.jar"
-  from sourceSets.main.output
+  classifier = 'fatjar'
+  from sourceSets.main.output.filter {
+    // This jar file needs to be excluded or else the shadow plugin will actually explode the classes into
```

Comment:
I don't have full context here, but this reads a bit confusing to me. We are excluding the file but want to include it (as a jar) in the output directory. Is it excluded entirely, or just marked for not being exploded into the fat jar?

Reply:
We are renaming it so that the shadow plugin doesn't extract all of the classes from it such that they live side-by-side with the toolkit classes.